webpack.config.js

```diff
@@ -0,0 +1,66 @@
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
+const MD5HashPlugin = require('webpack-md5-hash');
+const DefinePlugin = require('webpack/lib/DefinePlugin');
+const path = require('path');
+const { ENV, DEV, TEST, PROD } = require('./webpack.env');
+
+const outDir = path.resolve('bundle');
+const localComponents = /@storefront/;
+
+module.exports = {
+  entry: './presets/index',
+
+  devtool: 'source-map',
+
+  output: {
+    filename: `storefront.js`,
+    sourceMapFilename: `storefront.map.js`,
+    path: outDir,
+  },
+
+  resolve: {
+    // mainFields: ['browser', 'module:esnext', 'module', 'main']
+    alias: {
+      riot: path.resolve(__dirname, 'node_modules/riot/riot+compiler.js')
+    }
+  },
+  plugins: [
+    // new BundleAnalyzerPlugin(),
+    new MD5HashPlugin(),
+    new DefinePlugin({
+      'process.env': {
+        ENV: `'${ENV}'`,
+        NODE_ENV: `'${ENV}'`
+      }
+    }),
+  ].concat(ENV === PROD ? [new UglifyJSPlugin({ sourceMap: true })] : []),
+
+  module: {
+    rules: [{
+      test: path.resolve(__dirname, 'presets', 'core.js'),
+      use: { loader: 'expose-loader', options: 'storefront' }
+    }, {
+      resource: [
+        // /node_modules\/@storefront\/.*\.js$/,
```

Comment:
Is this required?

Reply:
This was in the file in storefront-presets, but it appears to be left over from an early iteration. I think we can safely remove it.

Reply:
Sounds good.